src/model.ts

```diff
@@ -102,7 +102,8 @@ export interface ModelConfig {
   readonly transportService: TransportService;
   readonly client: Client;
   readonly snapshotRequested: boolean;
-  readonly notifyUI: (payload: NotifyUIEvent) => void;
+  readonly notifyUI: (payload: InformationNotification) => void;
+  readonly raiseErrorUI: (text: string) => void;
```

Comment:
should probably be the same-ish interface for both. `raiseErrorUI: (payload :{ text :string})`